tests/conftest.py

```diff
@@ -36,6 +37,34 @@
 
 assert "airflow" not in sys.modules, "No airflow module can be imported before these lines"
 
+# Clear all Environment Variables that might have side effect,
+# For example, defined in /files/airflow-breeze-config/variables.env
+_AIRFLOW_CONFIG_PATTERN = re.compile(r"^AIRFLOW__(.+)__(.+)$")
+_KEEP_CONFIGS_SETTINGS: dict[str, dict[str, set[str]]] = {
+    # Keep always these configurations
+    "always": {
+        "database": {"sql_alchemy_conn"},
+        "core": {"sql_alchemy_conn"},
```

Comment:
I guess we also need to remove pre-2.3 configuration option, but keep it for a while, since it defined later in `conftest.py`

Reply:
Yeah. Likely revieweing and creating our own pytest plugin for those is a better idea for a follow up PR :)